apps/02RockPaperScissors.js

```diff
@@ -10,8 +10,30 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
```

Comment:
Even though it is obvious in this case, it is a good idea to put a comment explaining what each function does.
So you could put:
/*
* This function determines which hand won in the game "Rock, Paper, Scissors".
* 'rock' beats 'scissors', 'scissors' beats 'paper', and 'paper' beats 'rock'.
*/